### DISCUSSION TITLE
How to install x86_64 windows on Mac M1

### DISCUSSION THREAD

**Discussion by ivan2375** [utmapp/UTM#5089] [UTM Help (macOS)] [answered]
![Screen Shot 2023-03-04 at 01 29 19](https://user-images.githubusercontent.com/31252842/222801469-a9b1d016-6077-4fc8-99af-5b6f460286f2.jpg)
**Describe the issue**  
Someone mentioned that UTM can install x86_64 windows, so I bought this App.
But seems does not work.

**Configuration**  
* UTM Version: 4.1.6(75)
* macOS Version: 12.6.4
* Mac Chip (Intel, M1, ...): M1

**Comment by osy** [comment, marked as answer]
Provide your config.plist inside the .utm file. I’m guessing you did something weird outside of the wizard.

**Reply by ivan2375** [reply]
this config.plist is imported from vhd

<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>Backend</key>
	<string>QEMU</string>
	<key>ConfigurationVersion</key>
	<integer>4</integer>
	<key>Display</key>
	<array>
		<dict>
			<key>DownscalingFilter</key>
			<string>Linear</string>
			<key>DynamicResolution</key>
			<true/>
			<key>Hardware</key>
			<string>virtio-vga</string>
			<key>NativeResolution</key>
			<false/>
			<key>UpscalingFilter</key>
			<string>Nearest</string>
		</dict>
	</array>
	<key>Drive</key>
	<array>
		<dict>
			<key>Identifier</key>
			<string>86281AEB-6912-4859-9391-70712463487E</string>
			<key>ImageName</key>
			<string>SAP ECC6.0 EHP8.qcow2</string>
			<key>ImageType</key>
			<string>Disk</string>
			<key>Interface</key>
			<string>NVMe</string>
			<key>ReadOnly</key>
			<false/>
		</dict>
		<dict>
			<key>Identifier</key>
			<string>DABB2A48-0D28-4665-AD99-6F9212BAFF55</string>
			<key>ImageType</key>
			<string>CD</string>
			<key>Interface</key>
			<string>USB</string>
		</dict>
	</array>
	<key>Information</key>
	<dict>
		<key>Icon</key>
		<string>windows</string>
		<key>IconCustom</key>
		<false/>
		<key>Name</key>
		<string>SAP ECC6.0 EHP8</string>
		<key>UUID</key>
		<string>FB2EEA6F-CDF4-4EF5-8E51-51249F14089B</string>
	</dict>
	<key>Input</key>
	<dict>
		<key>MaximumUsbShare</key>
		<integer>3</integer>
		<key>UsbBusSupport</key>
		<string>3.0</string>
		<key>UsbSharing</key>
		<false/>
	</dict>
	<key>Network</key>
	<array>
		<dict>
			<key>Hardware</key>
			<string>e1000</string>
			<key>IsolateFromHost</key>
			<false/>
			<key>MacAddress</key>
			<string>B2:DC:84:31:16:F7</string>
			<key>Mode</key>
			<string>Shared</string>
			<key>PortForward</key>
			<array/>
		</dict>
	</array>
	<key>QEMU</key>
	<dict>
		<key>AdditionalArguments</key>
		<array/>
		<key>BalloonDevice</key>
		<false/>
		<key>DebugLog</key>
		<false/>
		<key>Hypervisor</key>
		<false/>
		<key>PS2Controller</key>
		<false/>
		<key>RNGDevice</key>
		<true/>
		<key>RTCLocalTime</key>
		<false/>
		<key>TPMDevice</key>
		<false/>
		<key>UEFIBoot</key>
		<true/>
	</dict>
	<key>Serial</key>
	<array/>
	<key>Sharing</key>
	<dict>
		<key>ClipboardSharing</key>
		<true/>
		<key>DirectoryShareMode</key>
		<string>WebDAV</string>
		<key>DirectoryShareReadOnly</key>
		<false/>
	</dict>
	<key>Sound</key>
	<array>
		<dict>
			<key>Hardware</key>
			<string>intel-hda</string>
		</dict>
	</array>
	<key>System</key>
	<dict>
		<key>Architecture</key>
		<string>x86_64</string>
		<key>CPU</key>
		<string>default</string>
		<key>CPUCount</key>
		<integer>4</integer>
		<key>CPUFlagsAdd</key>
		<array/>
		<key>CPUFlagsRemove</key>
		<array/>
		<key>ForceMulticore</key>
		<false/>
		<key>JITCacheSize</key>
		<integer>0</integer>
		<key>MemorySize</key>
		<integer>8192</integer>
		<key>Target</key>
		<string>q35</string>
	</dict>
</dict>
</plist>

------------------
This config.plist is trying to install a new machine from a windows 2008 iso file.
-------------------
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>Backend</key>
	<string>QEMU</string>
	<key>ConfigurationVersion</key>
	<integer>4</integer>
	<key>Display</key>
	<array>
		<dict>
			<key>DownscalingFilter</key>
			<string>Linear</string>
			<key>DynamicResolution</key>
			<true/>
			<key>Hardware</key>
			<string>virtio-vga</string>
			<key>NativeResolution</key>
			<false/>
			<key>UpscalingFilter</key>
			<string>Nearest</string>
		</dict>
	</array>
	<key>Drive</key>
	<array>
		<dict>
			<key>Identifier</key>
			<string>380B24D1-A517-4ACD-A203-EB5177E9E4A4</string>
			<key>ImageType</key>
			<string>CD</string>
			<key>Interface</key>
			<string>USB</string>
		</dict>
		<dict>
			<key>Identifier</key>
			<string>38501788-397C-43B7-8C2D-49E0AC09A2B0</string>
			<key>ImageName</key>
			<string>38501788-397C-43B7-8C2D-49E0AC09A2B0.qcow2</string>
			<key>ImageType</key>
			<string>Disk</string>
			<key>Interface</key>
			<string>NVMe</string>
			<key>ReadOnly</key>
			<false/>
		</dict>
		<dict>
			<key>Identifier</key>
			<string>2FB661F8-7206-4F92-832D-07DBE9932FCC</string>
			<key>ImageType</key>
			<string>CD</string>
			<key>Interface</key>
			<string>USB</string>
		</dict>
	</array>
	<key>Information</key>
	<dict>
		<key>Icon</key>
		<string>windows</string>
		<key>IconCustom</key>
		<false/>
		<key>Name</key>
		<string>SAP ECC6.0 EHP7</string>
		<key>UUID</key>
		<string>56CAE7F3-A8C2-49EE-8BE5-15D72D9EAC5A</string>
	</dict>
	<key>Input</key>
	<dict>
		<key>MaximumUsbShare</key>
		<integer>3</integer>
		<key>UsbBusSupport</key>
		<string>3.0</string>
		<key>UsbSharing</key>
		<false/>
	</dict>
	<key>Network</key>
	<array>
		<dict>
			<key>Hardware</key>
			<string>e1000</string>
			<key>IsolateFromHost</key>
			<false/>
			<key>MacAddress</key>
			<string>9E:AF:99:FC:50:A4</string>
			<key>Mode</key>
			<string>Shared</string>
			<key>PortForward</key>
			<array/>
		</dict>
	</array>
	<key>QEMU</key>
	<dict>
		<key>AdditionalArguments</key>
		<array/>
		<key>BalloonDevice</key>
		<false/>
		<key>DebugLog</key>
		<false/>
		<key>Hypervisor</key>
		<false/>
		<key>PS2Controller</key>
		<false/>
		<key>RNGDevice</key>
		<true/>
		<key>RTCLocalTime</key>
		<false/>
		<key>TPMDevice</key>
		<false/>
		<key>UEFIBoot</key>
		<true/>
	</dict>
	<key>Serial</key>
	<array/>
	<key>Sharing</key>
	<dict>
		<key>ClipboardSharing</key>
		<true/>
		<key>DirectoryShareMode</key>
		<string>WebDAV</string>
		<key>DirectoryShareReadOnly</key>
		<false/>
	</dict>
	<key>Sound</key>
	<array>
		<dict>
			<key>Hardware</key>
			<string>intel-hda</string>
		</dict>
	</array>
	<key>System</key>
	<dict>
		<key>Architecture</key>
		<string>x86_64</string>
		<key>CPU</key>
		<string>default</string>
		<key>CPUCount</key>
		<integer>4</integer>
		<key>CPUFlagsAdd</key>
		<array/>
		<key>CPUFlagsRemove</key>
		<array/>
		<key>ForceMulticore</key>
		<false/>
		<key>JITCacheSize</key>
		<integer>0</integer>
		<key>MemorySize</key>
		<integer>8192</integer>
		<key>Target</key>
		<string>q35</string>
	</dict>
</dict>
</plist>

**Reply by osy** [reply]
There isn't any support for importing vhd. You need to create a new VM using the wizard and a windows install ISO.

**Reply by ivan2375** [reply]
I have posted another plist, It is created by a new VM using the wizard to install windows by ISO.

**Reply by ivan2375** [reply]
It's got the same error also.

**Reply by osy** [reply]
Can you get a debug log?

For all issues related to running a VM, _including_ crashes, you should attach a debug log. (This is unavailable for macOS-on-macOS VMs. Attach an excerpt of your system log instead.)
To get the Debug log: open UTM, and open the settings for the VM you wish to launch. Near the top of the `QEMU` page is `Debug Log`. Turn it on and save the VM. After you experience the issue, open the VM settings again and select `Export Log...` and attach it here.

**Reply by ivan2375** [reply]
please see the attached.
[debug.log](https://github.com/utmapp/UTM/files/10887295/debug.log)

**Reply by osy** [reply]
Can you please zip and attach your .utm for the wizard created VM? It should not have any personal data as you have not installed windows.

**Reply by ivan2375** [reply]
Here is a zip file.
[SAP ECC6.0 EHP7.utm.zip](https://github.com/utmapp/UTM/files/10887308/SAP.ECC6.0.EHP7.utm.zip)

**Reply by osy** [reply]
So the issue is that your efi_vars.fd is from ARM UEFI and not from i386 UEFI. This is weird because the wizard should have picked the right file. Are you sure you didn't select ARM architecture in the wizard and the manually changed the architecture to x86_64? Did you change any settings after running the wizard?

**Reply by ivan2375** [reply]
Here is a picture captured from the screenshot.

![Screen Shot 2023-03-04 at 11 06 26](https://user-images.githubusercontent.com/31252842/222872845-95ee3f1a-5c65-484e-8099-48a57d0e8c4b.jpg)

**Reply by osy** [reply]
You didn't modify any settings after creating the VM with the wizard?

**Reply by ivan2375** [reply]
I did not modify any settings.

**Reply by osy** [reply]
Create a new VM using the wizard:
1. Emulate
2. Windows
3. Uncheck "Windows 10 or later"
4. Browse and select your iso
5. Continue
6. Continue
7. Continue
8. Continue
9. Save

This should create a working VM. Don't touch anything else.

**Reply by ivan2375** [reply]
Here is a test result that was following your step.

![Screen Shot 2023-03-04 at 11 33 20](https://user-images.githubusercontent.com/31252842/222873839-94a0a520-9e72-476d-93df-d4ae33272a37.jpg)
![Screen Shot 2023-03-04 at 11 34 05](https://user-images.githubusercontent.com/31252842/222873835-bc339f6b-0543-4aa8-bb84-25a440dd380e.jpg)

**Reply by osy** [reply]
That's not right, it's still showing "ARM64" as architecture. Are you sure you picked "Emulate" at the beginning? Are you sure you didn't change the architecture at the hardware screen? Maybe it'll be easier if you do a screen recording of you going through the wizard.

**Reply by ivan2375** [reply]
It's really appreciated, It's installing the windows server 2008. that is cause I chose the wrong type, not picked the "Emulate". But the running seems very slow.

**Reply by ivan2375** [reply]
Can I move the VM to another path? It does not has a function to choose the VM path in the create VM period.

**Reply by ivan2375** [reply]
It's so sorry, I have a question, Is't has not allowed to import the VM machine from another existing VM like VMware or parallels?

**Reply by osy** [reply]
https://github.com/utmapp/UTM/discussions/5089#discussioncomment-5199075
